py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -370,3 +372,108 @@ def index(
             reshape_output = reshape_layer.get_output(0)
 
     return reshape_output
+
+
+def scatter_value(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    dim: int,
+    index: Union[TRTTensor, np.ndarray, torch.Tensor],
+    value: float,
+) -> TRTTensor:
+    if not isinstance(input, TRTTensor):
+        raise RuntimeError(
+            f"scatter_tensor received input {input} that is not part "
+            "of the TensorRT region!"
+        )
+    input_shape = input.shape
+    index_shape = index.shape
+    index_shape_list = list(index.shape)
+    if not (isinstance(index, TRTTensor)):
+        index = get_trt_tensor(ctx, index, f"_index_tensor")
+    if len(input_shape) != len(index_shape):
+        raise RuntimeError(f"The no of dimensions of input and index should be equal")
+    dim = get_positive_dim(dim, len(input_shape))
+    dynamic_shape = has_dynamic_shape(input.shape)
+    if dynamic_shape:
+        # Check whether slice target dim is dynamic shape dim
+        assert input.shape[dim] != -1, "Can't scatter on negative shape dimension!"
```

Comment:
This should be checked in a converter validator, since it would otherwise cause a model failure

Reply:
Should it be a converter validator? Since a validator would be some feature of the converter such that it will be supported later? But this is not so in this case.

Reply:
In this case, even if there is no way to support the feature later, the check should still be in the validator so that we can fall back to Torch if this behavior is not supported